src/components/DataDuesAction/index.js

```diff
@@ -213,11 +213,11 @@ const DebtForm = ({
         />
       </Form.Group>
 
-      <Form.Group controlId={`interestRate${debtId}`}>
-        <Form.Label>Creditor</Form.Label>
+      <Form.Group controlId={`creditor${debtId}`}>
+        <Form.Label>{isStudentDebt ? 'Servicer' : 'Creditor'}</Form.Label>
         <Form.Control
           type="text"
-          placeholder="Sallie Mae"
+          placeholder={isStudentDebt ? 'Navient' : 'Sallie Mae'}
```

Comment:
Something to consider for future is that Placeholder doesn't get translated.
In that sense, we should use labels like material ones in order to have a _"better accessibility"_

Reply:
yes, good thing to have in mind. Thanks!